charts/opentelemetry-operator/README.md

```diff
@@ -158,6 +158,7 @@ metadata:
   name: my-collector
 spec:
   mode: daemonset
+  hostNetwork: true
```

Comment:
For future readers, this is necessary because the OpenTelemetryCollector object doesn't allow specifying hostPort. There is an open issue tracking the feature: https://github.com/open-telemetry/opentelemetry-operator/issues/1243